src/Wrep/Daemonizable/Command/EndlessCommand.php

```diff
@@ -97,8 +97,13 @@ protected function runloop(InputInterface $input, OutputInterface $output)
 	{
 		try
 		{
-			do
+			$this->starting($input, $output);
+
+			while (!$this->shutdownRequested)
```

Comment:
By changing this you change the behaviour of DaemonizableCommand, first is was guaranteed to run at least once. Now it could exit before starting the first iteration.

I think we shouldn't change this unless it fixes an important issue, there could be people relying on this out there and I don't want to break things without good reason.